.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -1,26 +1,34 @@
 ---
 name: Onboarding
 about: An onboarding checklist for folks joining the team!
-title: Onboarding checklist for {Github Username}
+title: Onboarding checklist for {GitHub Username}
 labels: ''
 assignees: ''
 
 ---
-
-### Welcome to the team  {GH USERNAME}!
+### Welcome to the team @{GitHub Username}!
 
 Here's a few pointers to get you started as well as a checklist to make sure you've got access to :all-the-things:!
 
 **For you**
-- [ ] [Join](https://github.com/GSA/GitHub-Administration#joining-the-gsa-enterprise-organization) GSA’s Enterprise Github org
 - [ ] Review the [project README](https://docs.google.com/document/d/1g8nYqYS_ifFlZB-DBgfeSoJRMB__EqWsmLnacyk-bDI/) in drive
-- [ ] Join Slack channels in the project README
+- [ ] Join the Slack channels in the project README
```

Comment:
Should we link the README?

Reply:
I didn't because it was linked on the row above and I didn't want to be redundant. We can add it if you feel strongly about it though! (This PR already got merged.)

Reply:
Oh right, duh, lol